src/main/java/com/codeborne/selenide/appium/commands/AppiumScrollTo.java

```diff
@@ -0,0 +1,86 @@
+package com.codeborne.selenide.appium.commands;
+
+import com.codeborne.selenide.SelenideElement;
+import com.codeborne.selenide.commands.ScrollTo;
+import com.codeborne.selenide.impl.WebElementSource;
+import io.appium.java_client.AppiumDriver;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.interactions.PointerInput;
+import org.openqa.selenium.interactions.Sequence;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.Optional;
+
+import static com.codeborne.selenide.appium.WebdriverUnwrapper.cast;
+import static java.time.Duration.ofMillis;
+import static java.util.Collections.singletonList;
+
+
+@ParametersAreNonnullByDefault
+public class AppiumScrollTo extends ScrollTo {
+
+  private AppiumDriver appiumDriver;
```

Comment:
Don't store webdriver instance as a field. 
`Command` instances may be shared between multiple calls and threads. Let it be just a local variable.

Reply:
Point noted and fixed.